.github/workflows/main.yml

```diff
@@ -28,4 +28,4 @@ jobs:
       with:
         java-version: 8
     - name: Run Maven Targets
-      run: mvn -Dtest=\!MutantSwarmExtensionTest package jacoco:report spotbugs:spotbugs checkstyle:checkstyle javadoc:javadoc coveralls:report --batch-mode --show-version --activate-profiles coveralls -Dcoveralls.skip=$COVERALLS_SKIP -DrepoToken=${{ secrets.COVERALLS_REPO_TOKEN }}
+      run: mvn -Dtest=\!MutantSwarmExtensionTest package jacoco:report spotbugs:spotbugs checkstyle:checkstyle javadoc:javadoc coveralls:report --batch-mode --no-transfer-progress --show-version --activate-profiles coveralls -Dcoveralls.skip=$COVERALLS_SKIP -DrepoToken=${{ secrets.COVERALLS_REPO_TOKEN }}
```

Comment:
That looks like it did the trick - the GH actions output is much less verbose now. Can you apply the same change (and the skipping of the MutantSwarmExtenstionTest) to https://github.com/HotelsDotCom/mutant-swarm/blob/master/.travis.yml#L9 please.